docs/programming-oak.md

```diff
@@ -477,6 +477,31 @@ handle for the _write_ half of a different channel, which is then used for
 responses &ndash; so the new Node has a way of _sending_ externally, as well as
 receiving.
 
+If two nodes communicate using Protocol Buffers, channel handles can be shared
+by encoding them in a message:
+
+<!-- prettier-ignore-start -->
+[embedmd]:# (../examples/injection/proto/injection.proto Protobuf /^message BlobStoreSender/ /^}$/)
+```Protobuf
+message BlobStoreSender {
+  oak.handle.Sender sender = 1 [(oak.handle.message_type) = ".oak.examples.injection.BlobResponse"];
+}
+```
+<!-- prettier-ignore-end -->
+
+The `message_type` extension specifies the fully qualified (with leading dot)
+name of the message passed over the channel.
```

Comment:
Maybe be a bit clearer about the type safety and which channel is which?  

> As well as indicating that a channel handle should be passed alongside a serialized message, the `message_type` extension also increases type safety by indicating the type of (serialized) messages that are expected to be sent over the channel referenced by the included handle.

Reply:
I have added some more details on type safety, thanks! 😄